.github/workflows/auto-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Automated Releases
+on: workflow_dispatch
+
+# This Workflow is triggered through the GitHub API:
+#   curl -X Post \
+#        -H "Authorization: Bearer <token>" \
+#        -d '{"ref":"master"}' \
+#        https://api.github.com/repos/simple-icons/simple-icons-pdf/actions/workflows/auto-release.yml/dispatches
+# Replacing <token> by a personal access token with scope `public_repo`
+
+jobs:
+  auto-release:
+    name: Automated release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Ensure we are checked out on the master branch
+          ref: master
+          # Ensure custom credentials are used when pushing
+          persist-credentials: false
+      - name: Update simple-icons
+        id: update
+        run: |
+          newVersion="$(node scripts/bump-simpleicons.js)"
+          echo ::set-output name=NEW_VERSION::$newVersion
+      - name: Regenerate icons
+        run: |
+          # Delete Existing PDFs
+          npm run clean
+          # Generate New PDFs
+          npm run build
+      - name: Commit updates
+        run: |
+          # Set up git credential
+          git config --global user.email "actions@github.com"
```

Comment:
Would be somewhat better to use 
```bash
git config --global user.email "github-actions[bot]@users.noreply.github.com"
git config --global user.name "github-actions[bot]"
```
A commit created with this information will be linked to https://github.com/apps/github-actions instead of a specific user on GitHub.

Current information will be linked to this user: https://github.com/actions-user (not sure who actually owns it)

Reply:
I like the suggestions @developStorm :+1:

If I recall correctly I actually copied the current approach from some official GitHub Actions example, I'd guess they (or at least someone) create the account for this purpose, but I think it's better to change it to your suggestion. Can you update this @adamrusted ?

Also, as I mentioned I originally wrote _this_ workflow for other simple-icons projects. So, if you want @devlopStorm, feel free to open Pull Requests against all simple-icons projects workflow files that make a commit like this.